corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -177,17 +200,15 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             this.model = this.options.model;
             this.errorMessage = null;
 
-            var value = this.model.get('value'),
+            // initialize with default values or with sticky values if either is present
+            var value = decodeValue(this.model, this.model.get('value'))[1],
                 allStickyValues = formplayerUtils.getStickyQueryInputs(),
                 stickyValue = allStickyValues[this.model.get('id')],
                 [searchForBlank, stickyValue] = decodeValue(this.model, stickyValue);
             this.model.set('searchForBlank', searchForBlank);
             if (stickyValue && !value) {  // Sticky values don't override default values
                 value = stickyValue;
             }
-            if ((this.model.get('input') === 'select' || this.model.get('input') === 'checkbox') && _.isString(value)) {
-                value = value.split(selectDelimiter);
-            }
```

Comment:
`decodeValue` does this already

Reply:
Slightly disappointed I didn't see this when adding the 'checkbox' check - good catch.

Reply:
I added `decodeValue` to this function in this PR to deal with the date stuff, so this was necessary previously :)